scripts/git_pr.sh

```diff
@@ -0,0 +1,8 @@
+./gradlew ktlintFormat
+git add -A && git commit -m "lint"
+if ./gradlew testDebugUnitTest
+then
+    gt stack submit
```

Comment:
```suggestion
    gt upstack submit
```

Reply:
Oh I guess both are valid commands, not sure which one is better for our needs.

Reply:
We can add the `-r` option to prompt for reviewers when creating the PR

Reply:
Alternatively, we can use the GitHub CLI, which contains more options and we know it will only submit the current branch

Reply:
https://cli.github.com/manual/gh_pr_create